src/Debugging/Drivers/RayDebugger.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Saloon\Debugging\Drivers;
+
+use Saloon\Debugging\DebugData;
+
+class RayDebugger implements DebuggingDriver
+{
+    public function name(): string
+    {
+        return 'ray';
+    }
+
+    /**
+     * @param \Saloon\Debugging\DebugData $data
+     *
+     * @return $this
+     */
+    public function send(DebugData $data): static
+    {
+        ray($this->formatData($data));
+
+        return $this;
+    }
+
+    /**
+     * @return array<string, mixed>
+     */
+    protected function formatData(DebugData $data): array
```

Comment:
Maybe move this method into the DebugData DTO so we don't need it on every driver, then if people need to format it further like for Telescope, they do it in the `send` method

Reply:
A DTO only carries the data, so I added the formatting methods on a base driver that all other drivers extend, instead.